Library/Koala/Koala.swift

```diff
@@ -2157,29 +2157,25 @@ private func properties(liveStreamEvent: LiveStreamEvent,
 }
 
 private func shareTypeProperty(_ shareType: UIActivity.ActivityType?) -> String? {
-  #if os(iOS)
-    guard let shareType = shareType else { return nil }
-
-    if shareType == .postToFacebook {
-      return "facebook"
-    } else if shareType == .message {
-      return "message"
-    } else if shareType == .mail {
-      return "email"
-    } else if shareType == .copyToPasteboard {
-      return "copy link"
-    } else if shareType == .postToTwitter {
-      return "twitter"
-    } else if shareType == UIActivity.ActivityType("com.apple.mobilenotes.SharingExtension") {
-      return "notes"
-    } else if shareType == SafariActivityType {
-      return "safari"
-    } else {
-      return shareType.rawValue
-    }
-  #else
-    return nil
-  #endif
+  guard let shareType = shareType else { return nil }
+
+  if shareType == .postToFacebook {
+    return "facebook"
+  } else if shareType == .message {
+    return "message"
+  } else if shareType == .mail {
+    return "email"
+  } else if shareType == .copyToPasteboard {
+    return "copy link"
+  } else if shareType == .postToTwitter {
+    return "twitter"
+  } else if shareType == UIActivity.ActivityType("com.apple.mobilenotes.SharingExtension") {
```

Comment:
Note-to-self to check if this is legit.

Reply:
Appears legit - https://stackoverflow.com/questions/31792506/how-to-exclude-notes-and-reminders-apps-from-the-uiactivityviewcontroller